docs/igor0806/GamesProject/calculator.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Basic JS</title>
+        <link rel="stylesheet" href="./styles/calculator.css">
+        <link rel="stylesheet" href="./styles/common.css">
+    </head>
+    <body>
+
+        <!-- Header  -->
+        <header class="intro">
+            <div class="container">
+                <div class="nav">
+                    <div class="logo">
+                        <img src="icon/logo.svg" alt="img">
+                    </div>
+
+                    <div class="nav-menu">
+                        <ul class="link_list">
+
+                            <li class="link">
+                                <a class="nav-link" href="./calculator.html">Calculator</a>
+                            </li>
+
+                            <li class="link">
+                                <a class="nav-link" href="./games.html">Games</a>
+                            </li>
+
+                            <li class="link">
+                                <a class="nav-link" href="./index.html">About</a>
+                            </li>
+
+                            <li class="link">
+                                <a class="nav-link" href="./form.html">Login</a>
+                            </li>
+
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <section class="calculator_main">
+            <div class="container">
+                <h1 class="calculator_title">Calculator v. 1</h1>
+                <div class="field">
+                    <label>
+                        <input id="firstValue" class="firstValue">
+                    </label>
+                    <label>
+                        <select name="operation" id="operation">
```

Comment:
You can add some styles for the select tag 
![2024-01-08_08-16](https://github.com/MastersAcademy/fe-course-basic-2023/assets/118203044/ace0d355-7f01-4ed8-949f-38a9ffeb7629)